src/schemas/user.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class User(BaseModel):
```

Comment:
```suggestion
class SignupRequest(BaseModel):
```

위처럼 API Request, Response 임을 클래스 이름에 명시해주는게 좀 더 명시적이고 시멘틱해보입니다.

Reply:
네 알겠습니다.